InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -384,6 +369,45 @@ class InMobiAdapter : PartnerAdapter {
         }
     }
 
+    /**
+     * Build a [JSONObject] that will be passed to the InMobi SDK for GDPR during [setUp].
```

Comment:
just a general note about comments: feel free to say what the method does. but as for usage, you have no control over it. sure, it currently is being used for gdpr during setup, but it _could_ be reused somewhere else.

Reply:
Noted. Thank you. 😃 👍🏼